core/templates/.spacemacs.template

```diff
@@ -252,8 +252,24 @@ values."
    ;; scrolling overrides the default behavior of Emacs which recenters point
    ;; when it reaches the top or bottom of the screen. (default t)
    dotspacemacs-smooth-scrolling t
+<<<<<<< afa2226051945b73c89fad46b56eb5fe6d4ece89
```

Comment:
I wonder why, but magit doesn't show these as an addition or deletion, but with the same color and face as the rest.
I must have missed because of something like that :smirk_cat: